### PR TITLE
Ubuntu/devel: New upstream snapshot 22.2-112-g78941a75-0ubuntu1~22.10.1

### PR DESCRIPTION
**Pull request by aciba90**
New upstream snapshot.
Do not squash.

SC-1208

### CODE REVIEW

**Comment by blackboxsw** [comment]
We discussed at standup we'd wait on the following PRs to land:
 - [✔DONE] https://github.com/canonical/cloud-init/pull/1609  
 - [in review] https://github.com/canonical/cloud-init/pull/1619

**Comment by blackboxsw** [comment]
#1619 merged. This branch and now update via `git fetch upstream; git reset --hard 7a973629e79303351113a7e6a20b9e0395585c61; new-upstream-snapshot` to re-perform all snapshot from scratch after your debian/gpb  changes

**Comment by aciba90** [comment]
@blackboxsw, updated with wanted commits and changelog redacted.

**Comment by blackboxsw** [comment]
manually redacted content in d/changelog looks good, version number looks good, I'll approve once I run things through sbuild and build-package/lintian testing locally

File: debian/changelog
Comment (blackboxsw) [inline debian/changelog:1]: made a minor change to version number in d/changelog, just force pushed  `22.2-115-g6e498773-0ubuntu1~22.10.1` instead of `22.2-115-g6e498773-0ubuntu1~22.10.1`

we don't need to `.2` because our official released version went from `22.2-64` to `22.2-115 in kinetic`

**Review by blackboxsw — APPROVED** [review]
- build-package and sbuild happy
- pkg version good at `22.2-115-g6e498773-0ubuntu1~22.10.1`
- only authorship changelog diffs and agreed manual redactions of testing/doc/non-ubuntu related commits
- awaiting jenkins test runners on ec2/azure with this content from daily build recipes. (lxd_container full integration run on kinetic success)

**Comment by blackboxsw** [comment]
Also force pushed a correction in the bug I incorrectly referenced  in my commit log https://bugs.launchpad.net/bugs/1982857 instead of https://bugs.launchpad.net/bugs/1958377

**Comment by blackboxsw** [comment]
Jenkins success runs w/ this functionality included in daily builds.
- lxd conainer: https://jenkins.canonical.com/server-team/view/cloud-init/job/cloud-init-integration-kinetic-lxd_container/50/ 
- Azure: https://jenkins.canonical.com/server-team/view/cloud-init/job/cloud-init-integration-kinetic-azure/51/
   * all green with exception of a an SDK timeout on tests/integration_tests/modules/test_set_password.py::TestPasswordListString, 
   * re-ran subset of tests on azure successfully
 ```
CLOUD_INIT_PLATFORM=azure CLOUD_INIT_OS_IMAGE=kinetic CLOUD_INIT_CLOUD_INIT_SOURCE=ppa:cloud-init-dev/daily tox -e integration-tests tests/integration_tests/modules/test_set_password.py::TestPasswordListString
...
============ 8 passed, 1 skipped, 49 warnings in 340.22s (0:05:40) =============
```
- ec2: https://jenkins.canonical.com/server-team/view/cloud-init/job/cloud-init-integration-kinetic-ec2/50/: SUCCESS

I'm uploading now to kinetic.

**Pull request merged**
